package.json

```diff
@@ -364,6 +493,18 @@
                     "when": "azurerm-vscode-tools-isParamFile",
                     "$when.comment": "Shows up when it's a param file, but only enabled if there is an associated template file"
                 }
+            ],
+            "view/title": [
+                {
+                    "command": "azurerm-vscode-tools.insertItem",
+                    "when": "azurerm-vscode-tools.template-outline.active == true",
```

Comment:
These are showing up in *all* custom views in vscode.  
<img width="368" alt="image" src="https://user-images.githubusercontent.com/6913354/80541473-0c08f200-8960-11ea-84fc-f86831300113.png">
I think what you want is:
"when": "view == azurerm-vscode-tools.template-outline"

Reply:
ditto for the inline ones, even though they don't currently show up because viewItem is filtering, but viewItem == "functions"/etc is not very unique and could still show up unintentionally.

Reply:
Fixed.